### ISSUE TITLE
AMECO Eviews add-in

### ISSUE THREAD

**Issue by RikkaConti** [xprimexinverse/EViews-Add-ins#1]
Hello Graeme,
I am very interested in your AMECO Eviews add-in, very practical!
It is taking me some time to have it download data past 2019 (as is needed now, EC's Spring Forecasts get to 2020).

Would you be so kind to show me the way?

Thank you very much,
Riccardo

**Comment by xprimexinverse** [comment]
Hi Riccardo,

Many thanks for getting in touch about this.

I updated the code for EViews to read in the 2020 data.

This technical issue was solved by changing the R code to use write.table() instead of write.csv2(). No other code changes were needed, thankfully, so a very straight forward fix. I updated the code with some comments for you to see what I done, if you're more curious to see how it was done.

Let me know it's successful for you now if you re-install the add-in.

Thanks!
Graeme
